### DISCUSSION TITLE
SABnzbd v4.1.0 released on Flathub

### DISCUSSION THREAD

**Discussion by francoism90** [sabnzbd/sabnzbd#2699] [General]
I've bumped and synced the Flathub/Flatpak SABnzbd package: https://github.com/flathub/org.sabnzbd.sabnzbd/commit/af110c9d020da95c181520271e385b6f8a957625

This also switches to `par2cmdline-turbo`, that's why I've created a discussion topic.

I've done some testing, and it seems to be working fine, and detect/use par2cmdline-turbo, but please let me know if it doesn't work or problems occur.

@Safihre it would be great if you could take a look at the commit as well. :)

**Comment by Safihre** [comment]
Seems good to me!

**Comment by francoism90** [comment]
@Safihre New release pushed.

Could you also please add a link to Flathub? So far only Snap is listed on the Downloads page. :)

**Reply by Safihre** [reply]
You can open an PR here to modify the website :)
https://github.com/sabnzbd/sabnzbd.github.io
